src/NLog/Targets/FileTarget.cs

```diff
@@ -1875,6 +1950,9 @@ private void DeleteOldArchiveFiles()
                     try
                     {
                         File.Delete(oldestArchivedFileName);
+                        var preventNullRefs = this.OnArchiveFileDeleted;
+                        if (preventNullRefs != null)
+                            preventNullRefs(oldestArchivedFileName);
```

Comment:
Code duplication. Should be call to `fileArchive_OnArchiveFileDeleted` / `fileArchive_OnFileArchived`?